### ISSUE TITLE
Séparation du problème et des solutions

### ISSUE THREAD

**Issue by nondidju** [progracqteur/WikiPedale#50]
Selon moi, la meilleure approche serait :
- L'énoncé du problème
  Apparaît en évidence
  Peut être édité par son auteur et un compte de type "associatif" ou "modérateur"
  Ne peut pas recevoir de commentaires
- Solutions
  Chaque solution est indépendante
  Tout le monde peut proposer une ou plusieurs solutions
  Une solution proposée par une administration ou un compte associatif doit être mis en évidence
  Chaque solution peut recueillir des votes et des commentaires
  La solution ayant recueilli le plus grand nombre de votes doit être mise en évidence
  Une photo ou un dessin peut être associé à chaque solution

Source d'inspiration : http://brainstorm.ubuntu.com

**Comment by julienfastre** [comment]
Si je comprends bien, tu proposerais qu'il y ait une partie "problème" pour chaque emplacement, et une partie "solution", c'est ça ? Et que l'on puisse voter et commenter chaque solution.

**Comment by nondidju** [comment]
Oui, c'est bien ça. Ça permettrait de mieux structurer les commentaires et de différencier les solutions officielles de celles proposées par les citoyens et associations.

**Comment by julienfastre** [comment]
Ca me semble être une proposition très intéressante, qui permettrait d'organiser les débats autrement que sur le site de La Meuse (pour avoir une référence :-) ). Et puis, ça inciterait les citoyens à proposer des solutions.

Il faudra veiller à bien l'organiser pour que ça reste ergonomique. 

Moi je suis pour.

Il reste à définir qui peut modifier quoi ? Pour moi, chaque personne non inscrite peut signaler un problème et commencer la description. Seuls les inscrits peuvent modifier la description. 

Pour les solutions: je propose que seuls les inscrits puissent proposer une solution et que, une fois entrée, seul un administrateur puisse la modifier (sinon il y aura des commentaires ou des votes sur des solutions qui seront modifiées après les commentaires ou les votes). Tous peuvent commenter et voter pour une solution.

**Comment by julienfastre** [comment]
Dans ce cas-là, j'imagine qu'on supprimerait l'issue #25 : les commentaires seraient des commentaires non plus sur les emplacements, mais plutôt sur la définition du problème et les solutions.

[edit] A moins que les commentaires ne portent que sur l'emplacement ? C'est peut-être moins casse-tête comme ça...

**Comment by marcu** [comment]
Je me demande si ce n'est pas trop réducteur. Il se peut que les gens veuillent poster des commentaires pour :
- dire qu'une partie du pbm a été traitée
- souligner un point qui n'a pas été soulevé
- ... 

Sans se limiter à "énoncé - solution", on peut imaginer mettre les commentaires "solution" en une autre couleur.

De plus certains points noirs n'ont pas besoin de solution (elle est évidente ou juste un manque d'entretien) :
http://orangeade.be/osm/points_noirs/index.php?id=49
http://orangeade.be/osm/points_noirs/index.php?id=82
http://orangeade.be/osm/points_noirs/index.php?id=41

Aussi les gens proposent des solutions dans l’énoncé du pbm :
http://orangeade.be/osm/points_noirs/index.php?id=82

**Comment by nondidju** [comment]
Il faudrait que chaque nouveau point noir soit modéré et ainsi :
- éviter que des solutions se retrouvent dans les énoncés (et dans ce cas, soit le modérateur sépare les deux, soit il invite à reformuler le point)
- si une partie du problème a été résolu, c'est qu'une des solutions a été activée et le commentaire peut s'y retrouver. Si le problème s'est résolu autrement, la solution peut être écrite à posteriori. On pourrait aussi imaginer une barre de progression (problème résolu à ... %) modifiable par l'auteur du problème ou un modérateur.
- certains énoncés peuvent ne pas avoir de solution, ce système ne les y oblige pas.

On peut aussi permettre les commentaires sur l'énoncé. J'ai peur qu'ainsi, le travail de modération soit plus important pour reclasser les commentaires aux bons endroits mais il est vrai que l'énoncé en lui-même peut être critiquable. On peut alors imaginer qu'un modérateur ait le droit de fermer les commentaires de l'énoncé lorsqu'il estime que le problème est suffisamment explicite (ou que cette décision ait été prise lors d'une réunion GRACQ par exemple). De même, on devrait pouvoir voter pour un énoncé pour en mesurer la pertinence (ou du moins l'attente du public) et séparément pour chaque solution afin d'en dégager celle qui semble la meilleure.

**Comment by julienfastre** [comment]
Je trouve intéressant qu'il y ait une séparation des problèmes et des solutions.

Pour le rabaissement de bordure, j'ai une fois eu une discussion avec un agent de la Ville. Il s'agissait ici d'une toute fine bordure de 2cm de haut qui allait être installée dans une rue, lors d'un réaménagement. Je craignais qu'elle soit aussi glissante qu'une autre de même type, installée ailleurs et je le lui ai dis. Il m'a expliqué qu'il estimait que ces deux centimètres étaient nécessaires pour empêcher le filet d'eau de remonter dans la rue qui allait être réaménagée. 

En discutant, on a trouvé une autre solution qu'il s'est promis d'étudier : décaler de quelques centimètres la bordure, de manière à ce que les vélos l'abordent de face, et non plus de côté quand elle était le long de la voirie. De cette manière on pouvait espérer qu'il y ait moins de problème. 

Je ne sais pas si cette solution est finalement la meilleure (on verra à l'usage), mais juste pour illustrer que parfois des problèmes qui ont l'air simple peuvent être résolus de différentes manières. 

(Une étude a été faite sur des classes d'enfants: les classes les moins violentes sont celles où les enfants peuvent proposer différentes solutions avant d'en choisir une ; les classes plus violentes sont celles où les enfants sont plus "pauvres" en solution. Pour cela, je pense que dans un esprit citoyen l'idée d'imaginer, ou de proposer aux utilisateurs d'imaginer, différentes solutions est intéressante ! )

Maintenant, je pense que les commentaires doivent être attachés à un emplacement, et pas à une solution. Sinon, ça m'apparait trop compliqué! C'est d'ailleurs le cas dans la référence de Benoit. 

On pourra en discuter par tél, évidemment.

**Comment by nondidju** [comment]
Ok pour n'appliquer les commentaires que sur l'énoncé du problème. On verra comment ça évolue et s'il faudra changer cela plus tard.

Ça m'intéresserait beaucoup de consulter l'étude que tu cites en exemple.

**Comment by julienfastre** [comment]
Je reporte à la milestone 1.5
